### ISSUE TITLE
Map in statistics is not filled with data

### ISSUE THREAD

**Issue by Marko-ilmkreis** [hzi-braunschweig/SORMAS-Project#4565]
### Bug Description

In statistics you have the option so get the data also shown on the map. Unfortunately now data were shown on the map.

### Steps to Reproduce
1. Go to statistics
2. Select the needed data and as visualization the map
3. the map stays empty

### Expected Behavior

All datas are shown on the map or the option for a map is hidden if its not possible to show the selected data.

### Screenshots

![grafik](https://user-images.githubusercontent.com/74791118/109466339-dc3be680-7a69-11eb-87ab-fc91d6954fff.png)

### System Details
* Device: Win10
* SORMAS version: 1.56.1
* Android version/Browser: FF

**Comment by Marc-FS** [comment]
Same here. Maybe insufficient User rights?
1.56.1

**Comment by Marko-ilmkreis** [comment]
No, its under my role. Admin, National user, National clinicain.

**Comment by JonasCir** [comment]
Hi @Marko-ilmkreis, did this issue came up after the 1.56.1 update?

**Comment by Marko-ilmkreis** [comment]
@JonasCir No, as far as I remember its in Sormas as long we are using it. In december the hotline told us that the developer are aware of it and going to fix it. An issue is not opened for that so far.

**Comment by JonasCir** [comment]
Hm at some point I managed to work around this, let's if I can remember it :D

**Comment by JonasCir** [comment]
@Marko-ilmkreis 
Hi, if you login as contact supervisor and navigate to the "Kontaktübersicht" (not statistics)

1. do you see anything there on the map? 
2. If you continue to navigate to the transmission chains ("Bitte vergrößern um die Infektionsketten zu sehen") do you see anything there?

**Comment by Marko-ilmkreis** [comment]
@JonasCir I tried it and it shows the data as expected for both (1&2).

**Comment by JonasCir** [comment]
Thanks, I think I identified the root [cause](https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/sormas-ui/src/main/java/de/symeda/sormas/ui/statistics/StatisticsView.java#L941):

The problem is that, the code tries to resolve shape files for the districts/regions given, however, for Germany there are no such files provided by default as it seems.

**Comment by Marko-ilmkreis** [comment]
Okay, that seems to be right. The map type can be selected, but this makes no sense for Germany anyway.
![grafik](https://user-images.githubusercontent.com/74791118/110314858-b59d2300-8008-11eb-8801-b9e3fae950cf.png)

**Comment by Marko-ilmkreis** [comment]
Okay, that seems to be right. The map type can be selected, but this makes no sense for Germany anyway as you don't have access to the instance of an other district.

![grafik](https://user-images.githubusercontent.com/74791118/110315024-ee3cfc80-8008-11eb-9d89-0b21e2a7f00a.png)

**Comment by JonasCir** [comment]
Alright :) Anyway, I'm going to find out if we should provide them and what's need to be done to create them for Germany.

**Comment by JonasCir** [comment]
We could use https://gdz.bkg.bund.de/index.php/default/digitale-geodaten/verwaltungsgebiete/verwaltungsgebiete-1-2-500-000-stand-01-01-vg2500.html 
as shapefiles for both German districts and regions @MartinWahnschaffe

**Comment by MartinWahnschaffe** [comment]
@JonasCir No objection with that. I guess we have to put the source note somewhere into the UI. Any suggestion?

**Comment by JonasCir** [comment]
Not sure but I think putting the disclaimer right below the map should work, there is already information displayed :)
Turned out that only adding the shapefiles is just half the rent, I have to refactor some code and testing is trickier than anticipated :D

**Comment by Marko-ilmkreis** [comment]
@JonasCir Thanks for your work!
